src/analytics.js

```diff
@@ -14,58 +14,127 @@ class Analytics {
       return false
     }
   }
+}
+
+class AnalyticsNode extends Analytics {
+  constructor (client) {
+    super(client)
+  }
+
+  // trackOpenDB (address, duration) {
+  //   let data = {}
+  //   data.event = 'open_db'
+  //   data.properties = { address: address, duration: duration }
+  //   this._track(data)
+  // }
 
-  trackOpenDB (address, duration) {
+  trackPinDB (odbAddress) {
     let data = {}
-    data.event = 'open_db'
-    data.properties = { address: address, duration: duration }
+    data.event = 'pin_db'
+    data.properties = { odb_address: odbAddress }
```

Comment:
We probably don't want to track the odbAddress but rather the hash of it?

Reply:
going to keep same as before for this PR, but yeah will likely switch to some form of that, but will wait to we can look at holistically across services